android/app/src/main/java/com/microsoft/codepush/react/CodePush.java

```diff
@@ -104,10 +104,6 @@ public CodePush(String deploymentKey, Activity mainActivity, boolean isDebugMode
             throw new CodePushUnknownException("Unable to get package info for " + applicationContext.getPackageName(), e);
         }
 
-        if (currentInstance != null) {
```

Comment:
i would prefer it if we left a comment describing how currentInstance could not be null but ultimately it doesn't matter